CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### New features
 
 * [#7274](https://github.com/rubocop-hq/rubocop/issues/7274): Add new `Lint/SendWithMixinArgument` cop. ([@koic][])
+* [#7295](https://github.com/rubocop-hq/rubocop/pull/7295): Add a plugin for `StyleGuideBaseURL`. ([@koic][])
```

Comment:
I think calling this a `plugin` might be a bit misleading - something like "Make it possible to set `StyleGuideBaseURL` per department." sounds better to me.

Reply:
Thanks for your review. I update it.